README.md

```diff
@@ -180,6 +184,11 @@ Table of Contents
 - [Amona: A protocol for private, asset-agnostic digital cash](https://anoma.net/) and their [use of recursive zkps](https://anoma.net/blog/demystifying-recursive-zkp/)
 - [ZETH: Integrating Zerocash on Ethereum](https://github.com/clearmatics/zeth)
 
+#### Confidential Transactions
```

Comment:
Nightfall could go to Layer 2 and Lelantus to Layer 1? They don't seem to be standalone applications or products, which may make them (and "Confidential Transactions" as a whole) stand out too much under the application category